content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -118,7 +118,7 @@ While `--apiserver-advertise-address` can be used to set the advertise address f
 control-plane node's API server, `--control-plane-endpoint` can be used to set the shared endpoint
 for all control-plane nodes.
 
-`--control-plane-endpoint` allows IP addresses but also DNS names that can map to IP addresses.
+`--control-plane-endpoint` allows not only IP addresses but also DNS names that can map to IP addresses.
```

Comment:
this is valid as per the "bot only … but also" sequence, but it think it makes the sentence a bit too pompous?

how about changing it to:
> `--control-plane-endpoint` allows both IP addresses and DNS names that can map to IP addresses.

/sig cluster-lifecycle
/kind cleanup

Reply:
yeah, this is easier to understand.